dashboard/src/features/shared/actions/create.js

```diff
@@ -7,6 +7,8 @@ export default function(type, options = {}) {
   const listPath = options.listPath || `/${type}s`
   const createPath = options.createPath || `${listPath}/create`
   const created = (param) => ({ type: `CREATED_${type.toUpperCase()}`, param })
+  const receivedKeys = (param) => ({ type: 'RECEIVED_MOCKHSM_ITEMS', param })
```

Comment:
This is a particularly tricky spot because the path we're in is `CREATED_ACCOUNT`, not `CREATED_MOCKHSM`. Creating an account has the occasional side effect of generating new Mock HSM keys.

Jeff, is your suggestion to dispatch `CREATED_MOCKHSM` here instead, and make sure the reducer handles that action appropriately? (i.e. a singular case of `RECEIVED_MOCKHSM_ITEMS`). As it stands, we're calling out to the vanilla Chain SDK here, so there's nothing for the reducer to do at this ponit.

An alternative would be dispatching `actions.mockhsm.create` as a thunk action here, which could then appropriately dispatch actions in its own flow, and return the Promise result after completion.